cleo_plugins/Audio/StreamHandle.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include "CLEO_Utils.h"
+#include "CAudioStream.h"
+
+using namespace CLEO;
+
+// scripts receive handles that are not exact pointers to the CAudioStream
+// instead adjusted address pointing "streamInternal" property is used, to mimick as it was in CLEO4
+class StreamHandle
+{
+    static const DWORD Handle_Offset = 4; // skip Virtual Method Table
```

Comment:
how can you guarantee that this offset is always 4?

Reply:
`offsetof` macro was returning 8 instead of 4. After investigation it turned out CLEO4 already had virtual methods in audio stream classes. Additional 4 bytes offset was caused by inheriting from VInjectible, which has VMT too.